gc/base/MemoryPoolLargeObjects.hpp

```diff
@@ -115,9 +117,42 @@ class MM_MemoryPoolLargeObjects : public MM_MemoryPool {
 	double resetTargetLOARatio(MM_EnvironmentBase*);
 	void resetLOASize(MM_EnvironmentBase*, double newLOARatio);
 	void redistributeFreeMemory(MM_EnvironmentBase*, uintptr_t newOldAreaSize);
-	bool isSizeEnoughForLOA(MM_EnvironmentBase* env, uintptr_t newOldAreaSize)
+
+	/* check if new LOA size is enough for largeObjectMinimumSize and set _loaSize, _soaSize, _currentLOARatio and _currentLOABase */
```

Comment:
Might want to use a doxygen comment
```suggestion
	/**
	 * Check if new LOA size is larger than largeObjectMinimumSize. If it is not large enough,
	 * set the LOA size to 0.  After, set the SOA size to be the remainder of the heap.
	 * @return If the newLOASize is large enough for the LOA
	 */
```